src/sloth256_189.cu

```diff
@@ -39,31 +39,34 @@ extern "C" bool detect_cuda()
     return cudaGetDeviceProperties(&prop, 0) == cudaSuccess;
 }
 
+#define STANDALONE_DEMO  // required if we want to compile .cu file, this is letting the below code become an 'entry' point
```

Comment:
Please remove this, it should be specified in compiler if at all needed

Reply:
Ah, I thought i did... Will do :/